docs/.vuepress/config.js

```diff
@@ -370,16 +370,13 @@ module.exports = {
     }
   },
   plugins: [
-    ['@vuepress/plugin-back-to-top', true],
+    '@vuepress/back-to-top',
```

Comment:
```suggestion
    '@vuepress/plugin-back-to-top',
```

`@vuepress/back-to-top` doesn't exist